packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -336,10 +344,11 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 			this.isMultiSortEnabled = StarkComponentUtil.isInputEnabled(this.multiSort);
 		}
 
-		if (changes["multiSelect"]) {
-			this.isMultiSelectEnabled = StarkComponentUtil.isInputEnabled(this.multiSelect);
-
-			if (this.isMultiSelectEnabled && this.displayedColumns) {
+		if (changes["multiSelect"] && this.isMultiSelectEnabled) {
+			if (!this.displayedColumns) {
+				this.displayedColumns = [];
```

Comment:
I changed the code in this block a bit, but there is still some logic about multiselect there. I could move it to a setter. But there would be some weird logic because it is a string treated as a boolean.
I'm watching for [this issue](https://github.com/angular/angular/issues/14761) but I doubt it will be picked up up anytime soon.

Reply:
Ah this is fine, I was referring to the fact that the `displayedColumns` were initialized here as well, but you have changed this already 👍